cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java

```diff
@@ -66,6 +68,13 @@ public Cocos2dxWebView(Context context, int viewTag) {
 
         this.setWebViewClient(new Cocos2dxWebViewClient());
         this.setWebChromeClient(new WebChromeClient());
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
cocos2d-x supports Android 2.3+, so we can not invoke WebView .setWebContentsDebuggingEnabled. I think should invoke by reflection. Or it will cause compiling issue.